src/srmouse/viz.py

```diff
@@ -42,6 +42,7 @@ def plotReflectanceSignature(df:pd.DataFrame, xlabel:str="Bands (nm)", ylabel:st
                bandNames)
     plt.title(title)
     plt.tight_layout()
+    # plt.savefig("Reflectance_Signature_results.png", dpi=150)  ## Uncomment this line, if you want to save plot as .png-file                           
```

Comment:
Add this option in the parameter of the methods for example you can set the paramter 

path2saveFigure:str=None 

and ask later 

if it is not None and is a valid string path save the figure